### DISCUSSION TITLE
How to set the default sync mode for a connection

### DISCUSSION THREAD

**Discussion by danielzepp** [airbytehq/airbyte#30783] [Connector Questions]
(I am using airbyte 0.50.30 OSS)

I created a connection to transfer data from PostgreSQL to Bigquery using CDC.
Even if a record is deleted in PostgreSQL, Bigquery expects the data to remain at the same time as the time is marked in "_ab_cdc_deleted_at".

From looking at the Airbyte documentation, I understand that if the sync mode is set to "Incremental | Append", it will work as I intended.

The point here is that there are about 500 tables in the database, and modifying the selection of all these tables by hand seems very difficult. (Also, I don't want to have human intervention every time a table is added.)
Is there a way to set the default sync mode to "Incremental | Append"?

**Comment by OrangeManLives** [comment]
Hi @danielzepp.
I also discovered that the bulk edit for sync mode had dissapeared after upgrading from Airbyte Version 0.40.28 to 0.50.34.

The bulk edit was removed here: https://github.com/airbytehq/airbyte/issues/26080.
I believe because it was broken.

This functionality is important to our use, we have around 30 DBs/Connections with around 600 tables.
Manually setting the sync mode per table is going to take forever.

You may be able to use the Airbyte API to Refresh the source schema and then potentially use this to to set the Sync Mode via the API but I havn't tried this yet.

[a comment by an automated account is not shown]
